Appraisals

```diff
@@ -1,7 +1,9 @@
-case(RUBY_VERSION)
+# frozen_string_literal: true
 
-when '2.5.8', '2.6.6', '2.7.1' then
-  appraise "capybara-3.0-ruby#{RUBY_VERSION}" do
-    gem 'capybara', '~> 3.0'
+case RUBY_VERSION
+when '2.7.7', '3.0.6', '3.1.3', '3.2.1'
```

Comment:
No need for `3.0.6` here. The other 3 are what we test in all other gems. We skipped over Ruby 3.0 and went to 3.1 in our applications.

Reply:
Since this is an open source gem and potentially used by the wider Ruby community we decided to add 30.x support

Reply:
Oh right, that makes sense. In that case you'll want to add it to the matrix in the CircleCI config.